src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -103,15 +106,18 @@ export class RundownTimelineService implements RundownService {
     this.stopAutoNext()
 
     const rundown: Rundown = await this.rundownRepository.getRundown(rundownId)
-
+    const snapBefore = rundown.getInfinitePiecesMap()
     rundown.takeNext()
     rundown.getActivePart().setEndState(this.getEndStateForActivePart(rundown))
 
     const timeline: Timeline = await this.buildAndPersistTimeline(rundown)
 
     this.startAutoNext(timeline, rundownId)
 
-    this.rundownEventEmitter.emitInfinitePiecesUpdatedEvent(rundown)
+    const snapAfter = rundown.getInfinitePiecesMap()
+    if (isDeepStrictEqual(snapBefore, snapAfter)) {
```

Comment:
Same comment about negation as before, am i mistaken or will we not just get an update event whenever nothing changes?

Reply:
ok, Now I see the 'negation' you are talking about here and above!